src/Rules/AllClasses.php

```diff
@@ -23,4 +26,11 @@ public function that(Expression $expression): AndThatShouldParser
 
         return new AndThatShould($this->ruleBuilder);
     }
+
+    public function exclude(string ...$classesToBeExcluded): ThatParser
```

Comment:
"All classes exclude Pippo that reside in one of these namespaces..."
doesn't sound like english... maybe we could substitute exclude with "except" or "a part of"?

Reply:
I think `except` is better.